crates/uv/tests/help.rs

```diff
@@ -19,6 +19,13 @@ fn help() {
       pip      Resolve and install Python packages
```

Comment:
We may want to add a better description here now that these are unhidden.

Reply:
I will follow